config/default/manager_auth_proxy_patch.yaml

```diff
@@ -0,0 +1,55 @@
+# This patch inject a sidecar container which is a HTTP proxy for the
+# controller manager, it performs RBAC authorization against the Kubernetes API using SubjectAccessReviews.
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
```

Comment:
Shall we change the names from `controller-manager` to `nats-manager` everywhere in code and yamls?